### PR TITLE
Add driverClass, driverMajorVersion and driverMinorVersion to AddJdbcDriverReque

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jmazzitelli could you please review?

File: hawkular-dmr-client/src/main/java/org/hawkular/dmrclient/DatasourceJBossASClient.java [hunk 1]
```diff
@@ -29,6 +30,7 @@
  * @author John Mazzitelli
  */
 public class DatasourceJBossASClient extends JBossASClient {
+    private static final Logger log = Logger.getLogger(DatasourceJBossASClient.class);
```
Comment (jmazzitelli) [inline hawkular-dmr-client/src/main/java/org/hawkular/dmrclient/DatasourceJBossASClient.java]: cant' you juse use the protected "log" member from JBossASClient - that's what its intended purpose was.

Comment (ppalaga) [inline hawkular-dmr-client/src/main/java/org/hawkular/dmrclient/DatasourceJBossASClient.java]: Call me stupid, I have not noticed that one. Let me fix it. Thanks

File: hawkular-dmr-client/src/main/java/org/hawkular/dmrclient/DatasourceJBossASClient.java [hunk 2]
```diff
@@ -152,33 +152,46 @@ public boolean isXADatasource(String datasourceName) throws Exception {
     /**
      * @param driverName the name of the JDBC driver to create, e.g. {@code "h2"}
      * @param moduleName the name of the WIldFly module that contains the driver jar file
+     * @return
```
Comment (jmazzitelli) [inline hawkular-dmr-client/src/main/java/org/hawkular/dmrclient/DatasourceJBossASClient.java]: empty return javadoc will fail javadoc verifier (if we ever run that). just put something in there after "return" to minimally describe it

**Pull request merged**